pkgs/by-name/un/unix-privesc-check/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "unix-privesc-check";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "http://pentestmonkey.net/tools/unix-privesc-check/${finalAttrs.pname}-${finalAttrs.version}.tar.gz";
+    hash = "sha256-4fhef2n6ut0jdWo9dqDj2GSyHih2O2DOLmGBKQ0cGWk=";
+  };
+
+  installPhase = ''
+    install -Dm 755 unix-privesc-check $out/bin/unix-privesc-check
```

Comment:
Also runHook preInstall and runHook postInstall

Reply:
Fixed